test('Hello world', ()=>{

})

test ('failed', () =>{
    throw new Error ('Failure')
})